src/client/components/app/Navigation/index.tsx

```diff
@@ -19,12 +19,11 @@ export interface NavigationLink {
 
 const StyledNavigation = styled.div``;
 
+interface NavigationProps {
+  hideOptionalLinks?: boolean;
+}
+
 const navLinks: NavigationLink[] = [
-  {
```

Comment:
We remove `/portfolio` from list? We still need it

Reply:
@Seroxdesign

Reply:
Resolved by returning the portfolio link and setting up flow to getUserPortfolio if portfolio/(no address)